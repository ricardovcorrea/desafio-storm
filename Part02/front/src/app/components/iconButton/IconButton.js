import React from "react";
import "./IconButton.scss";

const IconButton = () => {
  return <div></div>;
};

export default IconButton;
