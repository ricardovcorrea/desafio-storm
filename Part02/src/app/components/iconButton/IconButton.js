import React from "react";
import "./IconButton.scss";

import PropTypes from "prop-types";

import IconButton from "@material-ui/core/IconButton";
import Icon from "@material-ui/core/Icon";

const CustomIconButton = props => {
  return (
    <IconButton aria-label="delete">
      <Icon>{props.icon}</Icon>
    </IconButton>
  );
};

CustomIconButton.propTypes = {
  icon: PropTypes.string
};

export default CustomIconButton;
