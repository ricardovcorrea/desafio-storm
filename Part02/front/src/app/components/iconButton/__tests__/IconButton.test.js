import React from 'react';
import { render } from '@testing-library/react';
import IconButton from '../IconButton';

test('renders icon button', () => {
  const container = render(<IconButton />)
  expect(container.container).toMatchSnapshot()
});
