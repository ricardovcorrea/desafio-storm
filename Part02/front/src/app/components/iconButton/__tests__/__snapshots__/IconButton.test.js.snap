// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders icon button 1`] = `
<div>
  <div />
</div>
`;
